doc/tools/docgram/common.edit_mlg

```diff
@@ -714,10 +714,10 @@ gallina_ext: [
 
 | DELETE "Coercion" global ":" class_rawexpr ">->" class_rawexpr
 | REPLACE "Coercion" by_notation ":" class_rawexpr ">->" class_rawexpr
-| WITH "Coercion" smart_global ":" class_rawexpr ">->" class_rawexpr
+| WITH "Coercion" smart_global OPT [ ":" class_rawexpr ">->" class_rawexpr ]
```

Comment:
Think this is incorrect, should be reverted--you didn't change this production in the mlg file.

Reply:
In fact, I'd like
```
"Coercion" ident OPT [ ":" class_rawexpr ">->" class_rawexpr ]
"Coercion" ident OPT univ_decl def_body
```
but with the limitations of the parsing engine, I'm not able to write it that way in `g_vernac.mlg`, so I wrote there the equivalent
```
"Coercion" ident OPT [ OPT univ_decl def_body ]
"Coercion" ident ":" class_rawexpr ">->" class_rawexpr
```
Maybe we should present it in the doc as three versions of the command?
```
"Coercion" ident ":" class_rawexpr ">->" class_rawexpr  (* declare a coercion *)
"Coercion" ident OPT univ_decl def_body  (* combine definition and the previous *)
"Coercion" ident  (* edit an existing coercion *)
```

Reply:
Done that, hope this makes the thing more readable in the refman.

Reply:
Would you keep this as two entries?  We use multiple entries only when we're more or less forced into it, which is not the case here.  Then add some text explaining what it means if the class_rawexpr is not present.

Reply:
Sorry, what's unclear with "The third form..." just below the ".. cmd::"?

Reply:
We avoid using multiple entries unless we absolutely must.  We use 2 in a few places.  I'm not sure if we use 3 anywhere.